### DISCUSSION TITLE
AMP: url canonical dan url amphtml

### DISCUSSION THREAD

**Discussion by yuliakbard** [mecha-cms/mecha#196] [Q&A] [answered]
``` html
<link href="<?=From::mobileURL($url->clean); ?>" rel="canonical">
<?php if(!$site->is('mobile')):?>
<link href="<?= To::mobileURL($url->clean); ?>" rel="amphtml">
<?php endif;?>
```

kalau saya pake `$url-clean` pada url amphtml
outputnya:
`<link href="https://namadomain.com" rel="amphtml">`

kalau saya pake `$page->url` pada url amphtml
outputnya:
`<link href="https://namadomain.com/amp/sub-page" rel="amphtml">` (untuk kasus kalau sub-page nya kita jadikan sebagai halaman utama)

output yang saya harapkan itu:
`<link href="https://namadomain.com/amp" rel="amphtml">`

itu bagaimana ya kang?

**Comment by taufik-nurrohman** [comment, marked as answer]
Sudah Saya perbarui fungsinya mas.

**Reply by yuliakbard** [reply]
referensinya kang? tetap pake `$page->url` / `$url->clean` juga kah?

**Reply by taufik-nurrohman** [reply]
Di diskusi tentang AMP yang utama. Iya sama.
